test/cases/aws.sh

```diff
@@ -241,7 +240,14 @@ sudo "${CONTAINER_RUNTIME}" run \
     -e AWS_REGION="${AWS_REGION}" \
     -v "${TEMPDIR}":/tmp:Z \
     "${CONTAINER_CLOUD_IMAGE_VAL}" \
-    python cloud-image-val.py "${CIV_OPTIONS[@]}" && RESULTS=1 || RESULTS=0
+    python cloud-image-val.py \
+    ${CIV_OPTION_R:+"$CIV_OPTION_R"} \
+    ${CIV_OPTION_O:+"$CIV_OPTION_O"} \
+    ${CIV_OPTION_T:+"$CIV_OPTION_T"} \
+    ${CIV_OPTION_M:+"$CIV_OPTION_M"} \
+    ${CIV_OPTION_D:+"$CIV_OPTION_D"} \
+    ${CIV_OPTION_S:+"$CIV_OPTION_S"} \
+    && RESULTS=1 || RESULTS=0
```

Comment:
I don't understand what's the intention here? Do you mean to override the values of these variables based on externally defined values but have some sane defaults ? 

If that's the case I think you should move the defaults in the snippet above (or even have it unconditional) and simply use the values of these variables when calling `cloud-image-val.py`.

Reply:
Yeah I don't get this either, if you want defaults you can just put `${CIV_OPTION_R:-"my default value"} \` this will ensure that if you've set the variable sometime before it's going to use that but if you didn't set any value to it until now it will have the value of the string "my default value"

Reply:
I'm having the cloud-image-val options set into variables. This script is being executed both in osbuild-composer ci and cloud-image-val ci. I'm using this sintax as if I just used "${var}" the command would fail if var is empty

Reply:
There are no default options. If the script is being executed in osbuild-composer ci the options are set in the else clause in line 205. But not all the options are used all the time, so if an option is not set I don't want the command to fail

Reply:
It won't fail if you do something like this for example: `${CIV_OPTION_R:-} \` ... Same syntax was used on the now removed line 202

Reply:
But I have to quote it to get it passed shellcheck. Something like "${CIV_OPTION_R:-}". That when it fails